pkg/controller/compliancesuite/suitererunner.go

```diff
@@ -98,9 +124,11 @@ func GetRerunnerName(suiteName string) string {
 	return suiteName + "-rerunner"
 }
 
-func (r *ReconcileComplianceSuite) getRerunner(suite *compv1alpha1.ComplianceSuite) *batchv1beta1.CronJob {
+func (r *ReconcileComplianceSuite) getRerunner(suite *compv1alpha1.ComplianceSuite) (*batchv1beta1.CronJob, error) {
 	falseP := false
 	trueP := true
+	priorityClassName, err := r.getPriorityClassName(suite)
```

Comment:
As for right now, we are not validating if there is a priority class exists on the cluster, if the one reference is not there, it will just be ignored, when the pod is launched. I think I can add a warning if the priorityclass referenced was not there.

Reply:
I am going to add validation for this